### ISSUE TITLE
Min / Max Values at Colorbar

### ISSUE THREAD

**Issue by Meadril** [ScottPlot/ScottPlot#2297]
Hey, is it possible to set colours in the colormap/colourbar for values above the maximum value and below the minimum value? Currently there is only the possibility that the label changes to a larger smaller sign.
An example is the picture, if there is a value above 50 then the colour should be black, for example.

![image](https://user-images.githubusercontent.com/20774734/207327915-7a4bc98a-aaae-4a51-a7eb-7001bbf6d03f.png)

**Comment by swharden** [comment]
Hi @Meadril,

I don't think this is possible by definition. The colorbar uses a colormap to map a range of values to colors. If a value is outside that range, by definition it can't use logic to give it a special color (other than perhaps a color to indicate it's outside the range).

If you want a special colorbar (e.g., one that's rainbow from 0 to 50 but shades of gray from 50 to 100) you can achieve this by creating a custom colormap.

You can learn how colormaps work by inspecting the source code here:
https://github.com/ScottPlot/ScottPlot/tree/main/src/ScottPlot4/ScottPlot/Drawing/Colormaps

If you create your own colormap and assign it to your colorbar, you can probably achieve the effect you're looking for

If you come up with something that works well, I encourage you to revisit this issue and share your code with a screenshot so others can benefit from what you discover! 🚀

**Comment by Meadril** [comment]
![values](https://user-images.githubusercontent.com/20774734/210809366-c900e0ff-62ef-4e7a-b9b9-143028fc8e99.png)
![values2](https://user-images.githubusercontent.com/20774734/210809368-5e03c3c3-3229-43d5-b2e9-356f0e645345.png)

Hey, I have now found a quick solution for myself. But I will look for a more elegant solution as soon as I have time. Basically, I took a copy of a colormap and set the first and last values to white and black.
